plasma_framework/test/src/exits/payment/PaymentTransactionStateTransitionVerifier.test.js

```diff
@@ -106,7 +107,28 @@ contract('PaymentTransactionStateTransitionVerifier', ([alice, bob]) => {
         it('should return true for a valid state transition', async () => {
             const args = buildCorrectStateTransitionArgs();
             const verificationResult = await this.verifier.isCorrectStateTransition(
-                args.inFlightTx,
+                args.inFlightTxRaw,
+                args.inputTxs,
+                args.outputIndexOfInputTxs,
+            );
+            expect(verificationResult).to.be.true;
+        });
+
+        it('should not revert when number of transaction outputs is greater then number of transaction inputs', async () => {
```

Comment:
I saw the title of the PR and was somewhat confused. I'd suggest to rename things (PR and test here) more towards the behavior that we're expecting to see, along the lines of:

```
allows to start IFE with different configurations of inputs/outputs in teh transaction
```

where you would test a couple of situations (or all 25 if you prefer 0-4ins x 0-4outs :) )